ui/app/selectors/selectors.js

```diff
@@ -177,17 +177,17 @@ export function getAddressBookEntryName (state, address) {
 
 export function accountsWithSendEtherInfoSelector (state) {
   const accounts = getMetaMaskAccounts(state)
-  const { identities } = state.metamask
+  const identities = getMetaMaskIdentities(state)
 
   const accountsWithSendEtherInfo = Object.entries(identities).map(([key, identity]) => {
-    return Object.assign({}, accounts[key], identity)
+    return Object.assign({}, identity, accounts[key])
```

Comment:
The only property that will be overwritten here is `address`. The overwrite order was updated to match `getMetaMaskAccountsOrdered`.